object_storage_api/.env.example

```diff
@@ -19,6 +19,7 @@ OBJECT_STORAGE__ACCESS_KEY=root
 OBJECT_STORAGE__SECRET_ACCESS_KEY=example_password
 OBJECT_STORAGE__BUCKET_NAME=object-storage
 OBJECT_STORAGE__PRESIGNED_URL_EXPIRY_SECONDS=1800
+ATTACHMENT__ALLOWED_FILE_EXTENSIONS=[".doc", ".docx", ".pdf", ".txt"]
```

Comment:
From the meeting with CLF they suggested pdfs, excel spreadsheets and word documents. So looking at https://learn.microsoft.com/en-us/office/compatibility/office-file-format-reference I would add `.csv`, `.xls` and `.xlsx`. Maybe the more open formats `.odt`, `.rtf`, `.ods` as well? Then again I doubt they would use those so we can just add them if needed.

Reply:
one of the example attachments i added in https://github.com/ral-facilities/inventory-management-system/pull/1296 had a `.rtf` extension.

Reply:
I can add this, sure. Though, this is only an example file and the actual file will always be different depending on the facility.